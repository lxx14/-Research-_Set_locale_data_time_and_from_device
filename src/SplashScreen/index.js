import React from 'react';
import {
    View,
    Image
} from 'react-native';

import { styles } from './styles';

const SplashScreen = (props) => {

    React.useEffect(() => {
        setTimeout(
            goMainPage, 3000);
    }, []);

    goMainPage = () => {
        const { navigate } = props.navigation;
        navigate('MainApp');
    }

    return (
        <View style={styles.container}>
            <Image source={require('../assets/images/splash_logo.png')} />
        </View>
    )
}

export default SplashScreen;