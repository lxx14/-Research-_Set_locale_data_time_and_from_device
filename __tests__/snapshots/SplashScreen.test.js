import React from 'react';
import SplashScreen from '../../src/SplashScreen';
import renderer from 'react-test-renderer';

describe('SplashScreen', () => {
    it('create snapshot SplashScreen', () => {
        const snapshot = renderer.create(<SplashScreen />);
        expect(snapshot).toMatchSnapshot();
    });
})