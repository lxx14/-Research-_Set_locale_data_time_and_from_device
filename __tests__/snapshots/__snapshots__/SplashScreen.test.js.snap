// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplashScreen create snapshot SplashScreen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/splash_logo.png",
      }
    }
  />
</View>
`;
